### PR TITLE
google login setup completed

### PR DESCRIPTION
**Pull request by KrishnaTech6**
Here are some of the things/updates that I did 
- completed google login
- created profile activity with sign out button

### CODE REVIEW

File: .idea/misc.xml
Comment (Aditya13s) [inline .idea/misc.xml:1]: add all these ./idea files in .gitignore

File: build.gradle
Comment (Aditya13s) [inline build.gradle:1]: Don't Change Plugin Versions

File: app/build.gradle
Comment (Aditya13s) [inline app/build.gradle:1]: Please don't use Kotlin-android-extensions to access UI elements, it is actually deprecated now. Use View binding

File: build.gradle
Comment (KrishnaTech6) [reply build.gradle:1]: had to change , wasn't working

File: app/build.gradle
Comment (KrishnaTech6) [reply app/build.gradle:1]: ok

File: build.gradle
Comment (Aditya13s) [reply build.gradle:1]: Add a screenshot of the error

**Pull request closed without merging**